playbooks/02 - Use Case - Phishing Email Response/Investigate Suspicious Email.json

```diff
@@ -159,7 +159,7 @@
                 "SPF_Record": "{% set headers =  vars.input.records[0].emailHeaders | toDict %}{% if headers.header['received-spf'] -%}{{headers.header['received-spf']}}{% elif headers['received-spf'] -%}{{headers['received-spf']}}{%- endif %}",
                 "SuspiciousData": "['Bitcoins','bitcoin','torrent','limewire','bittottent','piratebay']",
                 "sender_return_path": "{{vars.input.records[0].returnPath}}",
-                "sender_email_address": "{{ vars.input.records[0].senderEmailAddress | regex_search('[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+') }}",
+                "sender_email_address": "{{ vars.input.records[0].senderEmailAddress | regex_search('[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+') }}",
```

Comment:
I tested this jinja in the playbook and it was working fine.
but in JSON format the escape char i.e. '\' char is represented using \\ that I missed while updating the JSON.

Reply:
You updated the json by hand? Why we didn't we export the PB?

Reply:
> I have added the regex to extract email from string in Investigate Phishing Email Response playbook
	
> Created Phishing email response Alert using Scenario 
	
> Executed Investigate Phishing Email Response playbook on Phishing Alert
	
> A playbook executed successfully with expected result
	
> Modified the "Investigate Phishing Email Response.json" file and replace the extract indicator jinja expression in json file

I was replaced that jinja in JSON file manually, because if we import the playbook and replace it in git then the diff is very huge thats why i added it manually